hack/e2e.sh

```diff
@@ -99,15 +101,16 @@ echo "SKIP_UP: $SKIP_UP"
 echo "SKIP_DOWN: $SKIP_DOWN"
 echo "KIND_DATA_HOSTPATH: $KIND_DATA_HOSTPATH"
 echo "KUBE_VERSION: $KUBE_VERSION"
+echo "DOCKER_IO_MIRROR: $DOCKER_IO_MIRROR"
 
-# https://github.com/kubernetes-sigs/kind/releases/tag/v0.6.0
+# https://github.com/kubernetes-sigs/kind/releases/tag/v0.6.1
```

Comment:
new images for v0.6.1 has kindnetd image preloaded, it speeds the bootstrap process of k8s cluster